addons/wowl/static/src/action_manager/action_manager.ts

```diff
@@ -204,9 +205,16 @@ export class ActionContainer extends Component<{}, OdooEnv> {
   static template = tags.xml`
     <div t-name="wowl.ActionContainer" class="o_action_manager">
       <t t-if="main.Component" t-component="main.Component" t-props="main.props" t-key="main.id"/>
-      <Dialog t-if="dialog.Component" t-props="dialog.dialogProps" t-key="dialog.id" t-on-dialog-closed="_onDialogClosed">
+      <Dialog t-if="dialog.Component and !dialog.props.dialogClass" t-props="dialog.dialogProps"
```

Comment:
I have the same concern in the debugManager branch: have a custom Dialog Class that specifically handles action in Dialog
I made something in the branch: https://github.com/odoo-dev/odoo/blob/4cc80847b8a9d6b27e0af32330a37345a5984c86/addons/wowl/static/src/action_manager/action_manager.ts#L209
I kinda think it is better than passing a prop...